### ISSUE TITLE
npm start fails to run on Windows

### ISSUE THREAD

**Issue by miladvafaeifard** [eggheadio-projects/redux-and-the-state-adt#1]
please fix it by this:
"start": "nodemon index.js -q -x \\"cls && node -r esm index.js\\"",

**Comment by evilsoft** [comment]
In the readme it is stated that it requires a POSIX system to run this repo. As I have not used a windows box in about 12 years, I really have no means to support and test it in that environment. Did Canonical give the windows people a POSIX shell that you can use?

EDIT: Also doing so would as suggested would move it from the target of POSIX or at least force the creation of a `cls` alias.

**Comment by miladvafaeifard** [comment]
Yes, I saw the notice in the readme, however, let me clarify that `nodemon index.js -q -x 'clear && node -r esm index.js'`, whatever it starts running `npm start` in POSIX system or windowx command on windows os will still fail to run because of the option "-x" that will only run under the main windows command. on the other hand, there is still the same issue that if `clear` is changed to 'cls'.
### the Error shown after running

> \'\'cls' is not recognized as an internal or external command,
> operable program or batch file.
